srcpkgs/linux5.5/files/x86_64-dotconfig

```diff
@@ -604,7 +604,7 @@ CONFIG_CPU_IDLE=y
 CONFIG_CPU_IDLE_GOV_LADDER=y
 CONFIG_CPU_IDLE_GOV_MENU=y
 CONFIG_CPU_IDLE_GOV_TEO=y
-CONFIG_CPU_IDLE_GOV_HALTPOLL=y
+# CONFIG_CPU_IDLE_GOV_HALTPOLL is not set
```

Comment:
why? this seems to improve VMs latency... read about it

Reply:
New in 5.4, fine with me.